app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container.rb

```diff
@@ -22,6 +24,33 @@ class ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreContainer
     end
   end
 
+  def self.params_for_create(_ems)
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :name       => 'name',
+          :id         => 'name',
```

Comment:
>     options[:bucket] = options.delete(:name) unless options[:name].nil?

We should confirm if these options are passed in as symbol or string keys in this hash

Reply:
@agrare Just confirmed they are being passed in as symbols:
{:name=>"0", :ems_id=>"87", :create_bucket_configuration=>{:location_constraint=>"ap-east-1"}, :emsType=>"ManageIQ::Providers::Amazon::StorageManager::S3", :parent_emstype=>"ManageIQ::Providers::Amazon::CloudManager"}